Makefile

```diff
@@ -1,59 +1,83 @@
-ifneq (, $(BUILDX_BIN))
-	export BUILDX_CMD = $(BUILDX_BIN)
-else ifneq (, $(shell docker buildx version))
-	export BUILDX_CMD = docker buildx
-else ifneq (, $(shell which buildx))
-	export BUILDX_CMD = $(which buildx)
-else
-	$(error "Buildx is required: https://github.com/docker/buildx#installing")
-endif
+PKG := github.com/docker/buildx
+VERSION ?= $(shell git describe --match 'v[0-9]*' --dirty='.m' --always --tags)
+REVISION ?= $(shell git rev-parse HEAD)$(shell if ! git diff --no-ext-diff --quiet --exit-code; then echo .m; fi)
+
+CGO_ENABLED ?= 0
+LDFLAGS ?= -s -w -X ${PKG}/version.Version=${VERSION} -X ${PKG}/version.Revision=${REVISION} -X ${PKG}/version.Package=${PKG}
```

Comment:
... because here, it will skip those if they're not set (i.e., it doesn't _append_ these to the existing ones set, correct?

Reply:
^^ ignore this one; I thought `/tmp/.ldflags` was a "generic" LDFLAGS for the platform, but it includes these